engine/runtime/src/main/scala/org/enso/compiler/context/SuggestionBuilder.scala

```diff
@@ -83,20 +83,22 @@ final class SuggestionBuilder[A: IndexedSource](val source: A) {
                   _
                 ) if !m.isStaticWrapperForInstanceMethod =>
             val typeSignature = ir.getMetadata(TypeSignatures)
-            val selfTypeOpt = typePtr match {
+            val (selfTypeOpt, isStatic) = typePtr match {
               case Some(typePtr) =>
-                typePtr
+                val selfType = typePtr
                   .getMetadata(MethodDefinitions)
                   .map(_.target.qualifiedName)
+                selfType -> false
               case None =>
-                Some(module)
+                Some(module) -> true
```

Comment:
doesn't this assume that only module methods are static? Because type methods also _can_ be.

Reply:
Good call. I missed the fact that the IR node has `isStatic` method. Updated.